package.json

```diff
@@ -38,9 +40,5 @@
     "node": ">=0.8.0",
     "npm": ">=1.2.10"
   },
-  "licenses": [
-    {
-      "type": "MIT"
-    }
-  ]
+  "license": "MIT"
```

Comment:
Updated to [new format](https://docs.npmjs.com/files/package.json#license) to prevent warnings on npm install.